system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -86,9 +86,9 @@ typedef long suseconds_t;
 #ifdef __EMSCRIPTEN__
 // For canvas transfer implementation in Emscripten, use an extra 10th control field
```

Comment:
Please update this comment to say "extra 11th control field", so that it is clear that the extra variable is at index [10], and not the second to last element.